internal/network/command_dns_forwarder.go

```diff
@@ -115,5 +114,5 @@ func printDnsForwarderTable(cmd *cobra.Command, forwarder networkingdnsforwarder
 		})
 	}
 
-	return table.PrintWithAutoWrap(false)
+	return table.PrintWithAutoWrap(true)
```

Comment:
I think this should be equivalent
```suggestion
	return table.Print()
```